app/services/digest_initiator_service.rb

```diff
@@ -16,7 +18,21 @@ def call
       digest_run.save!
     end
 
-    #enqueue the digest creation workers TBC
+    #TODO think about this being retried
```

Comment:
I don't think retry is too big a concern (if I understand correctly) as it'll only reach this point in an execution if there was not a DigestRun for date and range